KAERA/KAERA/Scenes/Writing/View/WritePickerVC.swift

```diff
@@ -132,18 +132,20 @@ class WritePickerVC: BaseVC {
     private func postWorryContent(postWorryContent: WorryContentRequestModel) {
         startLoadingAnimation()
         WriteAPI.shared.postWorryContent(param: postWorryContent) { [weak self] result in
-            guard let result = result, let _ = result.data else {
+            guard let result = result, let data = result.data else {
                 self?.presentNetworkAlert()
                 return
             }
             self?.stopLoadingAnimation()
-            if let writeVC = self?.presentingViewController {
+            // TODO: 리스폰스 데이터 worryDetail로 변환해 넘겨주고 updateUI 실행
+            if let writeNC = self?.presentingViewController, let NC = writeNC as? BaseNC {
                 UIView.animate(withDuration: 0.5, animations: {
                     self?.pickerViewLayout.alpha = 0
                 }) { [weak self] _ in
-                    // TODO: 고민 작성 후 해당 고민 상세로 이동
                     self?.dismiss(animated: false) {
-                        writeVC.dismiss(animated: true)
+                        let worryDetailVC = HomeWorryDetailVC(worryId: data.worryId, type: .digging)
+                        NC.setViewControllers([worryDetailVC], animated: true)
+                        NC.popToRootViewController(animated: true)
```

Comment:
NC.setViewControllers([worryDetailVC], animated: true)
코드에서 이미 navigationController의 스택을 worryDetailVC하나로 설정해서, 아래의 popToRootViewController가 없더라도 코드가 정상적으로 돌아가지 않을까하는 생각이 들었어요!

```suggestion
                        let worryDetailVC = HomeWorryDetailVC(worryId: data.worryId, type: .digging)
                        NC.setViewControllers([worryDetailVC], animated: true)
                        /// NC.popToRootViewController(animated: true)
```

위처럼 마지막 줄을 제거하면 문제가 발생할까요?

Reply:
오호 제가 생각했던거는 setViewContollers로 navigationControllers의 viewController들을 설정하고 popToRootViewController 메서드로 이동하는거라고 생각했었는데
공식문서를 보니 setViewContollers 실행시 설정된 top viewController로 이동까지 시켜주네요 popToRootVC 부분 삭제해서 실행해서 확인해보니 잘 작동합니다 👍 감사합니다
 popToRootViewController부분은 삭제하였습니다!